client/src/components/LoginSession/tests/LoginSession-test.js

```diff
@@ -4,58 +4,69 @@ import React from 'react';
 import LoginSession from '../LoginSession';
 import Enzyme, { shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import backend from 'lib/Backend';
 import MockDate from 'mockdate';
 
-jest.mock('@silverstripe/reactstrap-confirm', () => jest.fn().mockImplementation(
-    () => Promise.resolve(true)
-));
+jest.mock('@silverstripe/reactstrap-confirm', () => () => Promise.resolve(true));
```

Comment:
This will let you test what happens when the user says no to the confirmation

```suggestion
let confirmDialog = true;
jest.mock('@silverstripe/reactstrap-confirm', () => () => Promise.resolve(confirmDialog));
```